src/pages/api/logout.js

```diff
@@ -0,0 +1,8 @@
+import { deleteCookie } from 'cookies-next';
+
+export default function logout(req, res) {
+    if (req.method === 'POST') {
+        deleteCookie('token', { req, res });
+        res.status(200).redirect('/login');
```

Comment:
this will make a post request to /login, when you're redirecting, which would be a problem on deploy

Reply:
how to fix that?